templates/default/hdfs-site.xml.erb

```diff
@@ -649,4 +649,12 @@
     </description>
   </property>
 
+  <property>
+    <name>dfs.namenode.subtree-executor-limit</name>
```

Comment:
Does this affect all subtree operations or only the block reporting?

Reply:
dfs.namenode.subtree-executor-limit  is wrong property name. it is fixed now